cargo/cargo_build_script.bzl

```diff
@@ -1,9 +1,18 @@
+load("@bazel_tools//tools/cpp:toolchain_utils.bzl", "find_cpp_toolchain")
 load("@io_bazel_rules_rust//rust:private/rustc.bzl", "BuildInfo", "DepInfo", "get_cc_toolchain", "get_compilation_mode_opts", "get_linker_and_args")
 load("@io_bazel_rules_rust//rust:private/utils.bzl", "find_toolchain")
 load("@io_bazel_rules_rust//rust:rust.bzl", "rust_binary")
-load("@bazel_tools//tools/cpp:toolchain_utils.bzl", "find_cpp_toolchain")
 
-def _cargo_build_script_run(ctx, script):
+def _build_script_impl(ctx):
```

Comment:
```
def _cargo_build_script_impl(ctx):
```

nit

Reply:
I thought common practice for implementation macros was `def _{rule_name}_impl(ctx):` Is this not the case?

Reply:
I thought this was the toplevel impl, but that's a macro